metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -85,6 +86,16 @@ class OracleConfig(BasicSQLAlchemyConfig):
         description="The data dictionary views mode, to extract information about schema objects "
         "('ALL' and 'DBA' views are supported). (https://docs.oracle.com/cd/E11882_01/nav/catalog_views.htm)",
     )
+    # oracledb settings to enable thick mode and client library location
+    thick_mode: Optional[bool] = Field(
+        default=False,
+        description="Connection defaults to thin mode. Set to True to enable thick mode.",
+    )
+    lib_dir: Optional[str] = Field(
```

Comment:
this one may be renamed as `thick_mode_lib_dir` or similar, just to emphasize this is only relevant in case of enabling thick mode